content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -111,6 +111,59 @@ DATABASE_URL="mysql://root:randompassword@localhost:3306/mydb"
 
 </SwitchTech>
 
+<SwitchTech technologies={['*', 'planetscale']}>
+
+```prisma file=prisma/schema.prisma
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+Note that the default schema created by `prisma init` uses PostgreSQL as the `provider`. For PlanetScale, you need to use the `mysql` provider instead:
+
+```prisma file=prisma/schema.prisma highlight=2;edit
+datasource db {
+  provider = "mysql"
+  url      = env("DATABASE_URL")
+}
+```
+
+In this case, the `url` is [set via an environment variable](/concepts/components/prisma-schema#accessing-environment-variables-from-the-schema) which is defined in `.env`:
+
+```bash file=.env
+DATABASE_URL="mysql://janedoe:mypassword@planetscale-hostname/mydb?sslaccept=strict"
+```
+
+You now need to adjust the connection URL to point to your own database.
+
+The format of the connection URL for your database typically depends on the database you use. PlanetScale uses the MySQL connection URL format, which has the following structure (the parts spelled all-uppercased are _placeholders_ for your specific connection details):
+
+```no-lines
+mysql://USER:PASSWORD@HOST:PORT/DATABASE
+```
+
+Here's a short explanation of each component:
+
+- `USER`: The name of your database user
+- `PASSWORD`: The password for your database user
+- `PORT`: The port where your database server is running (typically `3306` for MySQL)
+- `DATABASE`: The name of the [database](https://dev.mysql.com/doc/refman/8.0/en/creating-database.html)
+
+For a database hosted with PlanetScale, the [connection URL](/reference/database-reference/connection-urls) might look similar to this:
```

Comment:
Here it might be worth to explain how and where you get that in the PlanetScale UI.

Reply:
I can do this, but I'm a bit wary of introducing too much coupling to the PlanetScale docs/UI - @tanberry do you have thoughts on this?

Reply:
Considering you are unclear how this really works (see comment below), explaining how to get a connection string for a branch seems worthwhile to describe (and also unprobable to change a lot on their side).